nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/util/PGPUtil.java

```diff
@@ -46,9 +46,8 @@
     public static final int BUFFER_SIZE = 65536;
     public static final int BLOCK_SIZE = 4096;
 
-    public static void encrypt(InputStream in, OutputStream out, String algorithm, String provider, int cipher, String filename,
-                               PGPKeyEncryptionMethodGenerator encryptionMethodGenerator) throws IOException, PGPException {
-
+    public static void encrypt(InputStream in, OutputStream out, String algorithm, String provider, int cipher, String filename, PGPKeyEncryptionMethodGenerator encryptionMethodGenerator) throws
+            IOException, PGPException {
         final boolean isArmored = EncryptContent.isPGPArmoredAlgorithm(algorithm);
```

Comment:
@alopresto - rather than continuing to add to this PR, it looks like @mcgilman merged almost everything in, I think, other than the version number bump. If there are other changes, I think your branch has veered enough away from master that you may want to start from master.